staging/src/k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1/apiservice.go

```diff
@@ -57,20 +58,27 @@ type APIServiceInterface interface {
 type aPIServices struct {
 	client  rest.Interface
 	clients []rest.Interface
+	te      string
```

Comment:
te --> tenant

Reply:
Agree, te is a bad naming. However, this is auto-generated code. I need to change to the code generator template to make the change, which will affect hundreds of files. 

Let me make this refactoring in another PR.